i18n/en-US.json

```diff
@@ -47,6 +47,9 @@
   "REPLUGGED_PLUGINS_UNINSTALL": "Uninstall {pluginCount, plural, =1 {plugin} other {plugins}}",
   "REPLUGGED_PLUGINS_UNINSTALL_SURE": "Are you sure you want to uninstall {pluginCount, plural, =1 {this plugin} other {these plugins}}?",
   "REPLUGGED_PLUGINS_VERSION": "Version",
+  "REPLUGGED_PLUGINS_GITHUB": "Github Repository",
```

Comment:
GitHub has a capital H. Or just remove “GitHub”--I think “Repository” should suffice.

Reply:
sure